introduction.md

```diff
@@ -0,0 +1,2 @@
+Hi there, I'm John from France, A Qt programmer, I love Git path in CodeSChool.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.